### ISSUE TITLE
The engine does not list  "En passant" as a valid move

### ISSUE THREAD

**Issue by Hejazi23** [ThiruDev50/genetom-chess-engine-flutter#1]
It seems that the game engine does not recognize "En passant" case or does not list it as a "valid" move at least

**Comment by ThiruDev50** [comment]
yeah @Hejazi23 , I will include in the next version (Not sure when, Since busy with other projects), If possible you can contribute. Much appreciated

**Comment by Hejazi23** [comment]
Thank you
